test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -1164,6 +1183,43 @@ UCS_TEST_P(test_ucp_sockaddr_destroy_ep_on_err, onesided_bidi_sforce) {
     one_sided_disconnect(sender(),   UCP_EP_CLOSE_MODE_FLUSH);
 }
 
+UCS_TEST_P(test_ucp_sockaddr_destroy_ep_on_err, create_and_destroy_immediately)
```

Comment:
can u pls document what the test is doing?

Reply:
done